docs/config.md

```diff
@@ -528,6 +528,28 @@ You can define aliases for commands, including their arguments. For example:
 aliases.l = ["log", "-r", "(main..@):: | (main..@)-"]
 ```
 
+If you would like to create an alias that's more complicated than a short form
+of a single jj command, you can use `jj util exec` in your alias to execute
+arbitrary external commands. This is analogous to `git`'s feature where
+executables in you `$PATH` starting with `git-*` can be called implicitly with
+`git *`. Here are some examples:
```

Comment:
I would really like to mention some caveats here, as some scripts may break `jj op undo` and can cause irreparable damage to the repo. 

Another point to mention here, that we may subsume this feature with an actual scripting language in the future.

Reply:
> I forgot to mention that the feature may be removed, added that now:
> 
> > This feature may be removed entirely in the future. It may be replaced by an embedded scripting language with limited capabilities for safety.

This can be simplified to "This feature may be removed or replaced by an embedded scripting language in the future".